25-test_eai_data/ascii_chain.pem

```diff
@@ -0,0 +1,53 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This must be moved to `test/recipes/25-test_eai_data/`

Reply:
Fixed.